Header      header
MsgBodypart head
MsgBodypart abdomen
MsgWing     left
MsgWing     right
MsgAux      aux
